src/main/java/uk/gov/ida/verifymatchingservicetesttool/scenarios/ScenarioBase.java

```diff
@@ -18,4 +30,28 @@
     public ScenarioBase(ApplicationConfiguration configuration) {
         this.configuration = configuration;
     }
+
+    protected Map<String, String> readEntityAsMap(Response response) {
+        return response.readEntity(new GenericType<Map<String, String>>() {{ }});
+    }
+
+    protected void validateMatchNoMatch(Response response) {
+        assertThat(response.getHeaderString("Content-Type"), is(APPLICATION_JSON));
+        assertThat(response.getStatus(), is(OK.getStatusCode()));
+
+        Map<String, String> result = readEntityAsMap(response);
+
+        assertThat(result.keySet(), is(new HashSet<String>() {{ add("result"); }}));
+        assertThat(result.get("result"), anyOf(is("match"), is("no-match")));
+    }
+
+    public void validateSuccessFailure(Response response){
+        assertThat(response.getHeaderString("Content-Type"), Is.is(APPLICATION_JSON));
+        assertThat(response.getStatus(), Is.is(OK.getStatusCode()));
+
+        Map<String, String> result = readEntityAsMap(response);
+
+        assertThat(result.keySet(), CoreMatchers.is(new HashSet<String>() {{ add("result"); }}));
+        assertThat(result.get("result"), CoreMatchers.anyOf(CoreMatchers.is("success"), CoreMatchers.is("failure")));
```

Comment:
Any reason we're not using the static imports here (like line 45)?

Reply:
No reason. Have not noticed it. Will fix now